### DISCUSSION TITLE
install app get error Could not install app

### DISCUSSION THREAD

**Discussion by wangshuai-007** [datalust/seq-tickets#1616] [Help]
I want to install `Seq.Input.HealthCheck`

About three days ago,It works well.

But, today; when I click install .
I got error like this:
```
Could not install app
Response status code does not indicate success: 302 (Redirect).
```

How can I fix this error ?or get more information?

Seq version is :2022.1.7449

**Comment by nblumhardt** [comment]
Hi @wangshuai-007,

The failure reported here will be between Seq and your NuGet server, which I'm assuming is NuGet.org.

It could be a transient issue; if it doesn't resolve in a few hours, taking a diagnostic report in Settings > Diagnostics > Download a full diagnostic report, searching it for the error, and sending that through to `support@datalust.co`, will help us point you in the right direction.

Hope this helps,
Nick
